bitwarden_license/bit-web/src/app/admin-console/providers/settings/account.component.ts

```diff
@@ -21,7 +21,7 @@ import { DialogService } from "@bitwarden/components";
   templateUrl: "account.component.html",
 })
 // eslint-disable-next-line rxjs-angular/prefer-takeuntil
-export class AccountComponent {
+export class AccountComponent implements OnDestroy {
```

Comment:
I didn't spot there's also an `ngOnInit` here:

```suggestion
export class AccountComponent implements OnDestroy, OnInit {
```